.lgtm

```diff
@@ -0,0 +1,3 @@
+approvals = 1
+pattern = "(?i)LGTM"
+self_approval_off = true
```

Comment:
Note that this has to be `false` since you will be the only maintainer of the package.